wikimedia/wikipedia-ios
# Add new Wikipedia languages, consolidate language configuration updates through a single command line utility

**Fixes Phabricator ticket:** 
https://phabricator.wikimedia.org/T239605

### Notes
Previously, there were three separate places to update language configs when new languages were added:

1. Manually in the language list
2. Automatically in the codemirror-config command line utility for codemirror
3. Automatically via a nodeJS script for the URL parsing and routing

After this PR, all three pieces were consolidated into the command line utility and the utility was renamed to "Update Languages" in the target list. This allows us to script these configuration changes in Swift instead of JS and run the script in one place in Xcode.

### Test Steps
1. Run "Update Languages" from the target list and verify it works on your machine and doesn't change the output
2. Verify adding, removing, re-ordering languages in the list still works
3. Verify URL routing still works as expected on different languages
